_studio/shared/umc/codec/h265_dec/include/umc_h265_headers.h

```diff
@@ -38,7 +38,8 @@ class HeaderSet
 public:
 
     HeaderSet(Heap_Objects  *pObjHeap)
-        : m_pObjHeap(pObjHeap)
+        : m_Header()
```

Comment:
```std::vector``` has default constructor that will be called without this change as well as with it. What was the warning from static analysis?

Reply:
@mgonchar , "'this->m_Header' is not initialized in this constructor"

Reply:
Ok. Generally I don't mind on merging this. But please change wording from "issue" to "warning", because initial code hadn't got any real issue with initialization.